service-base/src/main/java/org/eclipse/hono/service/tracing/SamplerProducer.java

```diff
@@ -195,18 +195,12 @@ Resource resource() {
      */
     @Singleton
     @Produces
-    @IfBuildProperty(name = "quarkus.opentelemetry.enabled", stringValue = "true")
-    Sampler sampler(final TracerRuntimeConfig tracerRuntimeConfig) {
-        final var samplerConfig = tracerRuntimeConfig.sampler;
-        if (samplerConfig.parentBased) {
-            LOG.warn("""
-                    'quarkus.opentelemetry.tracer.sampler.parent-based' set to 'true' - \
-                    custom Hono Sampler will not be applied to child spans""");
-        }
-        final String samplerName = Optional.ofNullable(otelConfig.getString(PROPERTY_OTEL_TRACES_SAMPLER))
-                .orElse(samplerConfig.samplerName);
+    @IfBuildProperty(name = "quarkus.otel.enabled", stringValue = "true")
+    Sampler sampler(final OTelRuntimeConfig otelRuntimeConfig) {
```

Comment:
The param name in the javadoc needs to be adapted.

Reply:
Done.

Reply:
Actually, I think `otelRuntimeConfig`, as you used earlier, was the better fitting name here.

Reply:
My bad, I changed the Javadoc attribute name instead now.